File: app/src/main/java/com/example/aleksei/reminderapp/model/NoteDatabase.java
package com.example.aleksei.reminderapp.model;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import android.arch.persistence.room.TypeConverters;

@Database(entities = {Note.class}, version = 1)
@TypeConverters({Converters.class})
public abstract class NoteDatabase extends RoomDatabase {
    public abstract NoteDao getNotesDao();
}

File: app/src/main/java/com/reminderapp/mvp/data/NoteDatabase.kt
package com.aleksei.reminderapp.mvp.data

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters

@Database(entities = [Note::class], version = 1)
@TypeConverters(Converters::class)
abstract class NoteDatabase : RoomDatabase() {
    abstract fun notesDao(): NoteDao
}